api-reference/media-api/get-versions-details.md

```diff
@@ -0,0 +1,114 @@
+# Get all versions of a file
+
+{% swagger baseUrl="https://api.imagekit.io" path="/v1/files/:fileId/versions" method="get" summary="Get file version details API" %}
+{% swagger-description %}
+Get all the file version details and attributes of a file.
+{% endswagger-description %}
+
+{% swagger-parameter in="path" name="fileId" type="string" required="true" %}
+The unique fileId of the uploaded file. `fileId` is returned in list files API and upload API.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="header" name="Authorization" type="string" required="true" %}
+base64 encoding of `your_private_api_key:`
+
+**Note the colon in the end.**
+{% endswagger-parameter %}
+
+{% swagger-response status="200" description="An array of file objects is returned." %}
+```javascript
+[
+    {
+        "fileId": "598821f949c0a938d57563bd",
+        "type": "file-version",
+        "name": "file1.jpg",
+        "filePath": "/images/products/file1.jpg",
+        "tags": ["t-shirt", "round-neck", "sale2019"],
+        "AITags": [
+            {
+                "name": "Shirt",
+                "confidence": 90.12,
+                "source": "google-auto-tagging"
+            },
+            /* ... more googleVision tags ... */
+        ],
+        "versionInfo": {
+                "id": "697821f849c0a938d57563ce",
+                "name": "Version 2"
+        },
+        "isPrivateFile": false,
+        "customCoordinates": null,
+        "url": "https://ik.imagekit.io/your_imagekit_id/images/products/file1.jpg?ik-obj-version=bREnN9Z5VQQ5OOZCSvaXcO9SW.su4QLu",
```

Comment:
`bREnN9Z5VQQ5OOZCSvaXcO9SW.su4QLu` is this a correct version Id? It has dot in between.

Reply:
Yes, it is correct.